### ISSUE TITLE
Daily accessibility scans in content-build need to alert Slack

### ISSUE THREAD

**Issue by meganhkelley** [department-of-veterans-affairs/va.gov-team#22455]
## Details 

There are daily accessibility scans (sitemap nightwatch tests) in the content-build repo that seem to be running correctly but are not alerting in Slack as they should be. 

## Acceptance Criteria 
- [ ] Scheduled a11y tests in content-build repo are running correctly
- [ ] Output of scheduled tests alerts in `#-daily-accessibility-scan` correctly

**Comment by timwright12** [comment]
Opened devops PR to update seed job: https://github.com/department-of-veterans-affairs/devops/pull/8853
Working branch `fix/22455-daily-a11y-scans` to fix docker issue

**Comment by timwright12** [comment]
This is fixed, tests are running on a schedule and alerting slack http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fcontent-build-integration-testing/detail/content-build-integration-testing/60/pipeline/57/
